src/exports.jl

```diff
@@ -16,6 +16,9 @@ export
   @GateName_str,
   randomparams,
 
+# circuits/noise.jl
+  applynoise,
```

Comment:
This seems mostly for use within `buildcircuit`, is there a need to export it?

Reply:
Yes, I'm exporting this because I'm adding functionalities to add custom noise (#196)